src/library-authoring/common/context/LibraryContext.tsx

```diff
@@ -40,6 +40,14 @@ export type LibraryContextData = {
   isCreateCollectionModalOpen: boolean;
   openCreateCollectionModal: () => void;
   closeCreateCollectionModal: () => void;
+  // "Create New Section" modal
+  isCreateSectionModalOpen: boolean;
+  openCreateSectionModal: () => void;
+  closeCreateSectionModal: () => void;
+  // "Create New Subsection" modal
+  isCreateSubsectionModalOpen: boolean;
+  openCreateSubsectionModal: () => void;
+  closeCreateSubsectionModal: () => void;
   // "Create New Unit" modal
   isCreateUnitModalOpen: boolean;
   openCreateUnitModal: () => void;
```

Comment:
@navinkarkera what do you think about merging these values and functions? All are containers and open the same modal

Reply:
Looks good. What are your concerns?

Reply:
@ChrisChV No, it looks good now. Initially, I did not want to replace toggle with state, but it looks fine.